features/storage/blockdevice/MBRBlockDevice.h

```diff
@@ -52,7 +52,8 @@ class MBRBlockDevice : public BlockDevice {
      *
      *  @param bd       Block device to partition
      *  @param part     Partition to use, 1-4
-     *  @param type     8-bit partition type to identify partition contents
+     *  @param type     8-bit partition type to identify partition contents.
```

Comment:
Would it not be better to reference something in our own documentation website?

Reply:
@Irit I am looking for suggestion copy the table from https://en.wikipedia.org/wiki/Partition_type#List_of_partition_IDs wiki page and  keep into our Mbed OS official documentation which gets referred by on description the _type_ parameter (edited)

Reply:
Can't just copy stuff from Wiki; it's theirs, not ours, and that's a pretty large chunk to include with a reference.
It's also not always the most respectable source. If you can find this in a more respectable place, you can link to it.